perun/postprocess/regression_analysis/run.py

```diff
@@ -94,7 +100,7 @@ def store_model_counts(analysis: list[dict[str, Any]]) -> None:
 @click.option(
     "--regression_models",
     "-r",
-    type=click.Choice(regression_models.get_supported_models()),
+    type=click.Choice(perun.utils.structs.postprocess_public.get_supported_models()),
```

Comment:
Why full name? This seems like automatic change.

Reply:
Great, thanks for the catch. Fixed.